byte-buddy-dep/src/main/java/net/bytebuddy/description/type/TypeDescription.java

```diff
@@ -7079,6 +7184,14 @@ public static String getName(Class<?> type) {
                     : name.substring(0, anonymousLoaderIndex);
         }
 
+        public static ForLoadedType of(Class<?> type) {
```

Comment:
This should probably just return a TypeDescription. The implementation should not be exposed.